src/flexmeasures_client/s2/control_types/FRBC/frbc_tunes.py

```diff
@@ -103,11 +106,24 @@ def __init__(
         self._valid_from_shift = valid_from_shift
 
         self._active_recurring_schedule = False
+        self._timers = dict()
+
+    def is_timer_due(self, name: str):
+        if self._timers.get(name, datetime.now() - timedelta(hours=1)) < datetime.now():
```

Comment:
[Follow-up from other comment]

Otherwise, what is the `timedelta(hours=1)` doing here?